### PR TITLE
Updating version of workloads

### PR DESCRIPTION
**Pull request by anjalirai-intel**

### CODE REVIEW

File: data/tests_without_attestation.yaml [hunk 1]
```diff
@@ -24,53 +24,53 @@ test_redis_unknown_docker_image:
...
 
 test_redis_with_test_sign_key_debug:
-  docker_image: redis redis:7.0.0
+  docker_image: redis redis:7.0.10
   signing_key_path : test
   debug_mode: y
 
 test_redis_with_runtime_variables:
-  docker_image: redis redis:7.0.0
+  docker_image: redis redis:7.0.1000
```
Comment (jinengandhi-intel) [inline data/tests_without_attestation.yaml:40]: Is this a typo?

File: data/tests_without_attestation.yaml [hunk 2]
```diff
@@ -24,53 +24,53 @@ test_redis_unknown_docker_image:
...
 test_redis_ev_dict:
-  docker_image: redis redis:7.0.0
+  docker_image: redis redis:7.0.10
   runtime_variable_list: {name="Xyz",age="20"}
   expected_screen: docker_flags
   expected_output_console: 'Invalid string specified'
 
 test_redis_ev_invalid_input_2:
-  docker_image: redis redis:7.0.0
+  docker_image: redis redis:7.0.100000
```
Comment (jinengandhi-intel) [inline data/tests_without_attestation.yaml:73]: Is this a typo?

File: data/tests_without_attestation.yaml (hunk 1 above)
Comment (anjalirai-intel) [reply data/tests_without_attestation.yaml]: Done

File: data/tests_without_attestation.yaml (hunk 2 above)
Comment (anjalirai-intel) [reply data/tests_without_attestation.yaml]: Done

**Review by jinengandhi-intel — APPROVED** [review]

**Pull request merged**